internal/operators/rx_test.go

```diff
@@ -55,22 +55,42 @@ func TestRx(t *testing.T) {
 			want:    true,
 		},
 		{
-			// Requires multiline
+			// Requires multiline disabled by default
 			pattern: `^hello.*world`,
 			input:   "test\nhello\nworld",
+			want:    false,
+		},
+		{
+			// Makes sure multiline can be enabled by the user
+			pattern: `(?m)^hello.*world`,
+			input:   "test\nhello\nworld",
 			want:    true,
 		},
 		{
-			// Makes sure, (?sm) passed by the user does not
+			// Makes sure, (?s) passed by the user does not
 			// break the regex.
-			pattern: `(?sm)hello.*world`,
+			pattern: `(?s)hello.*world`,
 			input:   "hello\nworld",
 			want:    true,
 		},
 		{
 			// Make sure user flags are also applied
-			pattern: `(?i)^hello.*world`,
-			input:   "test\nHELLO\nworld",
+			pattern: `(?i)hello.*world`,
+			input:   "testHELLO\nworld",
+			want:    true,
+		},
+		{
+			// The so called DOLLAR_ENDONLY modifier in PCRE2 is meant to tweak the meaning of dollar '$'
+			// so that it matches only at the very end of the string (see: https://www.pcre.org/current/doc/html/pcre2pattern.html#SEC6)
+			// It seems that re2 already behaves like that by default.
+			pattern: `123$`,
```

Comment:
I was not able to find the `DOLLAR_ENDONLY` modifier for RE2, but it seems that re2 by default behaves like DOLLAR_ENDONLY enabled. 
See also https://regex101.com/r/mUcy69/1 (Mind the regex option D (dollar end only) enabled).
Am I missing something?